README.md

```diff
@@ -53,14 +53,23 @@ Add the following to your `tmux.conf` file to configure the tmux side of
 this customization.
 
 ``` tmux
-# Smart pane switching with awareness of vim splits
+# Smart pane switching with awareness of Vim splits.
 # See: https://github.com/christoomey/vim-tmux-navigator
-is_vim='echo "#{pane_current_command}" | grep -iqE "(^|\/)g?(view|n?vim?x?)(diff)?$"'
-bind -n C-h if-shell "$is_vim" "send-keys C-h" "select-pane -L"
-bind -n C-j if-shell "$is_vim" "send-keys C-j" "select-pane -D"
-bind -n C-k if-shell "$is_vim" "send-keys C-k" "select-pane -U"
-bind -n C-l if-shell "$is_vim" "send-keys C-l" "select-pane -R"
-bind -n C-\ if-shell "$is_vim" "send-keys C-\\" "select-pane -l"
+
+has_vim="ps -o state= -o comm= -t '#{pane_tty}' \
+    | grep -iEq '^[^TXZ ]+ +(\S+\/)?g?(view|n?vim?x?)(diff)?$'"
+tmux bind-key -n C-h  if-shell "$has_vim" "send-keys C-h"  "select-pane -L"
+tmux bind-key -n C-j  if-shell "$has_vim" "send-keys C-j"  "select-pane -D"
+tmux bind-key -n C-k  if-shell "$has_vim" "send-keys C-k"  "select-pane -U"
+tmux bind-key -n C-l  if-shell "$has_vim" "send-keys C-l"  "select-pane -R"
+tmux bind-key -n C-\\ if-shell "$has_vim" "send-keys C-\\" "select-pane -l"
```

Comment:
I was able to sort this out. Looks like tmux escaping the format string (specifically the `\S+\/` was being stripped down to `S+/`). The final form should be:

``` tmux
is_vim="ps -o state= -o comm= -t '#{pane_tty}' \
    | grep -iqE '^[^TXZ ]+ +(\\S+\\/)?g?(view|n?vim?x?)(diff)?$'"
bind-key -n C-h  if-shell "$is_vim" "send-keys C-h"  "select-pane -L"
bind-key -n C-j  if-shell "$is_vim" "send-keys C-j"  "select-pane -D"
bind-key -n C-k  if-shell "$is_vim" "send-keys C-k"  "select-pane -U"
bind-key -n C-l  if-shell "$is_vim" "send-keys C-l"  "select-pane -R"
bind-key -n C-\\ if-shell "$is_vim" "send-keys C-\\" "select-pane -l"
```

Note, this also removes likely accidental additions of `tmux` prefixing each command (likely a carryover from TPM config?).

Also, I'd like to stick to the `is_vim` nomenclature as I think it better captures the intent (especially with the addition of support for a backgrounded Vim session thanks to you @blueyed), and I'd prefer not to change that if we don't have a strong reason.